engine/env/spark/src/main/java/org/datacleaner/spark/SparkJobContext.java

```diff
@@ -220,11 +232,49 @@ public String getResultPath() {
         }
         return null;
     }
-    
+
+    /**
+     * Gets the job name (removing the extension '.analysis.xml')
+     * 
+     * @return
+     */
     public String getAnalysisJobName() {
         final int lastIndexOfSlash = _analysisJobXmlPath.lastIndexOf("/");
         final int lastIndexOfFileExtension = _analysisJobXmlPath.lastIndexOf(".analysis.xml");
-        final String jobName = _analysisJobXmlPath.substring(lastIndexOfSlash+1, lastIndexOfFileExtension);
-        return  jobName;
+        final String jobName = _analysisJobXmlPath.substring(lastIndexOfSlash + 1, lastIndexOfFileExtension);
+        return jobName;
+    }
+
+    /**
+     * Gets the hdfs path of job's result
+     * 
+     * @return
+     */
+    public String getResultJobFilePath() {
```

Comment:
the concept "result job file" does not make sense to me. Either it's a "job file" or a "result file".